workspaces/Cargo.toml

```diff
@@ -11,7 +11,7 @@ Library for automating workflows and testing NEAR smart contracts.
 
 [dependencies]
 async-trait = "0.1"
-async-process = { version = "1.3.0", optional = true }
+async-process = "1.3"
```

Comment:
Why is this not optional anymore? Doesn't seem like you've removed the unstable feature config from the only code that uses it?

Reply:
I am guessing because the new version of near-sandbox-utils pulls it in anyway

Reply:
oh wait, was originally gonna swap over to the async version, but then that was gonna be a huge process due to async drop and stuff. So this was remnant of that. It should be optional again since its no longer being used

Reply:
I don't think that's a good reason to import it again if not needed directly, is it?

Reply:
yeah sorry, I didn't mean to say that was the reason, but meant I'll remove it

Reply:
I think Austin was referring to my comment :)

Reply:
> I think Austin was referring to my comment :)

I was, for some reason my comments don't update in real-time :D 

not a big deal either way